docs/DeveloperGuide.md

```diff
@@ -486,367 +498,250 @@ Hence, any modification to an employee after SudoHR is initialized from storage
+**MSS**
 
+1. User requests to remove an employee from a given department.
+2. SudoHr removes the employee from the given department.
 
-**Use case: UC12 - Update event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to update event details on SudoHR
-4. SudoHR update event.
-5. Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+- 1a. Given employee does not exist.
 
-  Use case ends.
+  - 1a1. SudoHr shows an error message.
 
-* 3a. The given index is invalid.
+    Use case resumes at step 1.
 
-    * 3a1. SudoHR shows an error message.
+- 1b. Given department does not exist.
 
-      Use case resumes at step 2.
+  - 1b1. SudoHr shows an error message.
 
-* 3b. The given argument is empty.
+    Use case resumes at step 1.
 
-    * 3b1. SudoHR shows an error message.
+- 1c. The given employee does not belong to the given department.
 
-      Use case resumes at step 2.
+  - 1c1. SudoHr shows an error message.
 
+    Use case resumes at step 1.
 
-**Use case: UC13 - List event**
-**MSS:**
-1. User requests SudoHR to list all events
-2. SudoHR shows all events and their details
+**Use case: Listing all employees in a given department**
 
+1. User requests to list all employees in a given department.
+2. SudoHr lists all the employees in the given department.
 
-**Use case: UC14 - Add employee to event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to list employees
-4. SudoHR shows all employees and their details
-5. User requests to add employee to event on SudoHR
-6. SudoHR adds employee to event.
-7. Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 2a. The event list is empty.
-
-  Use case ends.
-
-* 4a. The employee list is empty.
-
-  Use case ends.
-
-* 5a. The given Employeeindex is invalid.
+- 1a. The given department does not exist.
 
-    * 5a1. SudoHR shows an error message.
+  - 1a1. SudoHr shows an error message.
 
-      Use case resumes at step 4.
+    Use case resumes at step 1.
 
-* 5b. The given Eventindex is invalid.
+- 1b. There are no employees in the given department.
 
-    * 5b1. SudoHR shows an error message.
+  Use case ends.
 
-      Use case resumes at step 4.
+**Use case: Listing all departments that an employee is in**
 
-**Use case: UC15 - List employees in an event**
-**MSS:**
-1. User requests to __list events(UC6)__
-2. SudoHR shows all events and their details
-3. User requests to __list employees in a event(UC8)__
-4. SudoHR shows all employees added to an event and their details
-5. Use case ends.
+1. User requests to list all the departments that a given employee is in.
+2. SudoHr lists all the department that the given employee is in.
 
 **Extensions**
 
-* 2a. The event list is empty.
+- 1a. The given employee does not exist.
+  - 1a1. SudoHr shows an error message.
+- 1b. The employee is not in any department.
```

Comment:
Since SudoHR still lists 0 departments, technically step 2 would be reached so this extension would not be needed.